### DISCUSSION TITLE
How do you avoid the HTTPS only behavior of the desktop? For example when visiting a localhost website?

### DISCUSSION THREAD

**Discussion by Julianoe** [goastian/midori-desktop#87] [Q&A]
Love seeing a webkit based browser for the linux desktop. Useful for web developers to test some things. The problem is I can't look at my websites locally because they are all broken and load no files because I don't have https:// locally. Is there a way around this? Someone already had a solution for it?

**Comment by Ponchale** [comment]
normally all you have to do is write http://localhost it works for all our staff.
